submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,177 @@
+const url = 'https://randomuser.me/api/?results=12';
+
+let aToZLast = document.querySelector('.sortedAtoZLast');
+let zToALast = document.querySelector('.sortedZtoALast');
+let atoZFirst = document.querySelector('.sortedAtoZFirst');
+let ztoAFirst = document.querySelector('.sortedZtoAFirst');
+let content = document.querySelector('.articleBlock');
+let reset = document.querySelector('.reset');
+let male = document.querySelector('#male');
+let woman = document.querySelector('#female');
+let all = document.querySelector('#all');
+let search = document.querySelector('#searchBar');
+let minNumber = document.querySelector('#minNumber');
+let maxNumber = document.querySelector('#maxNumber');
+
+let basicArr1 = [];
```

Comment:
`basicArr1` doesn't seem like a correct name. You should avoid using data type in the name of the variables, it indicates that the name is not descriptive enough. Like `resultArr` could just be `friends` or `users`. Plural name assumes that this is an array by itself.